keyboards/naked60/readme.md

```diff
@@ -0,0 +1,18 @@
+# naked60
+
+![naked60](https://cdn-ak.f.st-hatena.com/images/fotolife/S/Salicylic_acid3/20190530/20190530040355.jpg)
+
+This is 60 keys modification Ortholinear keyboard. 
+
+Keyboard Maintainer: [Salicylic_acid3](https://github.com/Salicylic-acid3)  
+Hardware Supported: The PCBs, controllers supported  
+Hardware Availability: links to where you can find this hardware
```

Comment:
Is there an actual link?

Reply:
Like Naked48, it uses completely different firmware on the left and right.
Therefore, Split Keyboard feature cannot be used.

The Readme has been updated.